src/main/java/com/sign/infrastructure/jpa/CertificationLog.java

```diff
@@ -0,0 +1,24 @@
+package com.sign.infrastructure.jpa;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
[사소한 제안]

롬복 어노테이션과 JPA 어노테이션의 순서 컨밴션을 지키는건 어떨까요? 해당 객체의 성격이 강한게 가장 안쪽으로 들어오게요 😄 
```suggestion
@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Entity
```

Reply:
요근래 코틀린만 쓰다가 자바 + JPA 쓰니까 이런 부분이 또 눈에 밟히는군요.. 이 부분은 조금 생각해봐야 할 것 같습니다. 별도 이슈로 빼서 컨벤션 맞춰보는건 어떨까요?